flink-core/src/main/java/org/apache/flink/api/java/typeutils/runtime/TupleSerializer.java

```diff
@@ -150,7 +150,7 @@ public T deserialize(T reuse, DataInputView source) throws IOException {
 	
 	private T instantiateRaw() {
 		try {
-			return tupleClass.newInstance();
+			return tupleClass.getDeclaredConstructor().newInstance();
```

Comment:
this change may have performance implications as noted in the mail thread linked in the jira.